docssrc/src/commandarguments.md

```diff
@@ -0,0 +1,202 @@
+# CommandArguments
+
+The `CommandArguments` class was introduced in CommandAPI 9.0.0 and provides a much more powerful way of accessing arguments than just an array of arguments which existed until 9.0.0.
+
+While the argument array just gives the possibility to access the arguments via the array notation (`args[0]`), the `CommandArguments` class offers much more, including:
+
+- [Access the inner structure directly](#access-the-inner-structure-directly)
+- [What terms are used?](#what-terms-are-used)
+  - [`nodeName`](#nodename)
+  - [`raw argument`](#raw-argument)
+  - [`unsafe argument`](#unsafe-argument)
```

Comment:
This doesn't make sense in the context of the sentence above it. The sentence above states "... class offers much more, including:" and then this bullet point says "What terms are used?".

It looks like you're trying to make a table of contents. This isn't necessary, assuming the table of contents renders properly on the side of the page (which may not because the code to do that is super old and was when I didn't know how to write JavaScript):

<img width="846" alt="image" src="https://github.com/JorelAli/CommandAPI/assets/4613171/c24536d7-a60f-4999-a749-a2d6c7ee07b3">

It may be better to just summarize the top-level sections, e.g:

- Accessing arguments by index (previous behavior)
- Accessing arguments by their node name
- Accessing raw arguments
- Accessing arguments with inferred type-checking